===================================================================
RCS file: /cvs/gnome/gthumb/components/catalog-view/Makefile.am,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- gthumb-2.6.8/components/catalog-view/Makefile.am	2005/08/07 10:51:41	1.6
+++ gthumb-2.6.8.new/components/catalog-view/Makefile.am	2005/11/15 09:13:05	1.7
@@ -47,15 +47,15 @@
 server_DATA = GNOME_GThumb_CatalogView.server
 GNOME_GThumb_CatalogView.server.in: GNOME_GThumb_CatalogView.server.in.in Makefile
 	sed -e "s|\@LIBEXECDIR\@|$(libexecdir)|g" \
-	< $(srcdir)/GNOME_GThumb_CatalogView.server.in.in > temp \
-	&& mv temp GNOME_GThumb_CatalogView.server.in
+	< $(srcdir)/GNOME_GThumb_CatalogView.server.in.in > GNOME_GThumb_CatalogView.server.in.temp \
+	&& mv GNOME_GThumb_CatalogView.server.in.temp GNOME_GThumb_CatalogView.server.in
 
 @INTLTOOL_SERVER_RULE@
 
 GNOME_GThumb_CatalogView.xml: GNOME_GThumb_CatalogView.xml.in Makefile
 	sed -e 's?\@ICONSDIR\@?$(datadir)/gthumb/icons?g' \
-	< $(srcdir)/GNOME_GThumb_CatalogView.xml.in > temp \
-	&& mv temp GNOME_GThumb_CatalogView.xml
+	< $(srcdir)/GNOME_GThumb_CatalogView.xml.in > GNOME_GThumb_CatalogView.xml.temp \
+	&& mv GNOME_GThumb_CatalogView.xml.temp GNOME_GThumb_CatalogView.xml
 
 uidir = $(datadir)/gnome-2.0/ui
 ui_DATA = GNOME_GThumb_CatalogView.xml
